quix-frontend/client/src/lib/ui/components/SortableTable.scss

```diff
@@ -0,0 +1,9 @@
+.bi-table-cells {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.bi-table-cell-query {
```

Comment:
This class doesn't belong to this generic table component. It should be moved to the component that defines the `query` column.